src/platform/i386/boot_comp.c

```diff
@@ -12,6 +12,9 @@
 
 extern u8_t *boot_comp_pgd;
 
+void *thd_mem[NUM_CPU], *tcap_mem[NUM_CPU];
+struct captbl *ct;
```

Comment:
Perhaps use `glb_boot_ct` as the variable name here? It's confusing with the rest of the functions which use `ct` though from it's local scope.

Reply:
indeed.